### PR TITLE
Allow only one primary rate limit for response headers

### LINKED ISSUE #1684: Rate limiting response_headers should work like radio button
Left as enhancement:
User should be able to select only one **'show rate limit in response headers'** value when multiple rate limit rules are defined. This is used as the primary `response_headers` value on API Umbrella, and there can be only one primary rule.

nayttokuva 2016-10-03 kello 17 59 02

### PR DESCRIPTION
**Pull request by jykae**
Closes #1684 

- onChange event handler to allow only one primary rate limit for response headers

### CODE REVIEW

File: proxy_backends/client/form/form.js [hunk 1]
```diff
@@ -156,4 +159,19 @@ Template.apiProxy.events({
       });
     }
   },
+  'change input:checkbox[name*="response_headers"]': (event) => {
+    const changedCheckbox = $(event.target);
+
+    const checkboxesArray = $("input:checkbox[name*='response_headers']");
```
Comment (marla-singer) [inline proxy_backends/client/form/form.js]: Please add a comment for this line to describe an operation

**Comment by marla-singer** [comment]
@jykae Works good. It's almost ready to merge

**Comment by jykae** [comment]
@marla-singer comment added

**Comment by marla-singer** [comment]
@jykae Merge it

**Pull request merged**